### DISCUSSION TITLE
i cannot migrate

### DISCUSSION THREAD

**Discussion by fajar7xx** [spatie/laravel-activitylog#756] [Q&A]
i have installed this package. i followed the instruction after publish i try to migrate and its fails 

the error is 

`SQLSTATE[42000]: Syntax error or access violation: 1103 Incorrect table name '' (SQL: create table `` (`id` bigint unsigned not null auto_increment primary key, `log_name` varchar(255) null, `description` text not null, `subject_id` bigint unsigned null, `subject_type` varchar(255) null, `causer_id` bigint unsigned null, `causer_type` varchar(255) null, `properties` json null, `created_at` timestamp null, `updated_at` timestamp null) default character set utf8mb4 collate 'utf8mb4_unicode_ci')`

**Comment by Gummibeer** [comment]
Hey,

the missing table name is most common a problem with a cached config.
You should run `artisan config:clear`.

https://twitter.com/devgummibeer/status/1277950600437813248
